velox/dwio/common/Options.h

```diff
@@ -462,6 +465,14 @@ class ReaderOptions {
     return *this;
   }
 
+  /**
+   * Modify the number of row groups to prefetch.
```

Comment:
Please fix the code style as https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#comments

Here we should use //